components/charts/TextStats.vue

```diff
@@ -2,27 +2,55 @@
   <v-container>
     <!-- Days you are chatting -->
     <v-row class="my-16 pa-8 white--text">
-      <v-col class="cyan darken-2" style="position: relative" cols="12" sm="6">
-        <v-icon size="100">mdi-calendar</v-icon>
-        <v-col cols="12" class="text-h5 font-weight-bold pa-0 ma-0 text-center"
-          >You chatted for
-        </v-col>
-        <div class="text-h1 font-weight-bold ma-auto my-0 pa-4">
-          {{ dateDiffs }}
-        </div>
-        <v-col cols="12" class="text-h5 font-weight-bold pa-0 ma-0 text-center"
-          >days
-        </v-col>
+      <v-col class="cyan darken-2 fact-box py-10" cols="12" sm="6">
+        <v-icon v-show="$vuetify.breakpoint.mdAndUp" size="100"
```

Comment:
maybe just put it in the same row as the number?
or have on col left with the icon
and then col to the right with all the text -> no overlap
or just have the icon in the background?
i like the icons, so kind a sad that they are gone on mobile

Reply:
hm I ll try